### PR TITLE
[ICDS Dashboard] Add ui element tabular report

### PR DESCRIPTION
**Pull request by Rohit25negi**
This code change is adding the UI element for the new tabular report Currently backend data code is not written for this so returned the dummy response. To save time rolled out this. 
This should not be a problem because it is under the Feature flag and not accessible to external users.
Ticket: https://app.asana.com/0/1160310200927866/1157414140082847
under FF: Yes

### CODE REVIEW

File: custom/icds_reports/tasks.py [hunk 1]
```diff
@@ -920,9 +921,15 @@ def prepare_excel_reports(config, aggregation_level, include_test, beta, locatio
             )
         else:
             cache_key = create_excel_file(excel_data, data_type, file_format)
-
+    elif indicator == SERVICE_DELIVERY_REPORT:
+        # CODE IN THIS SECTION WILL BE CHANGED WITH ACTUAL DATA PULLING
+        # WHEN THE BACKEND DATA WILL BE READY FOR THIS TABLE. MEANWHILE THIS
+        # REPORT IS NOT accessible to the USER BECAUSE ITS IN FF
```
Comment (czue) [inline custom/icds_reports/tasks.py]: i like how you stopped shouting for a few words :upside_down_face:

File: custom/icds_reports/templates/icds_reports/icds_app/download.directive.html [hunk 2]
```diff
@@ -56,6 +56,21 @@
                 </div>
             </div>
             <div ng-if="!$ctrl.isISSNIPMonthlyRegisterSelected()">
+
+                <div class="form-group" ng-if="($ctrl.isSDRSelected())">
+                    <label class="control-label col-sm-2" for="format">Beneficiary: <i class="fa fa-info-circle" uib-popover="Select the category of beneficiaries"
```
Comment (czue) [inline custom/icds_reports/templates/icds_reports/icds_app/download.directive.html]: nit: should this be "select the category of *beneficiary*"?

File: custom/icds_reports/static/icds_reports/js/spec/download.directive.spec.js [hunk 3]
```diff
@@ -588,7 +588,7 @@ describe('Download Directive', function () {
 
         it('tests that block user does not have access to dashboard usage report', function () {
             var length = controller.indicators.length;
-            assert.equal(10, length);
+            assert.equal(11, length);
```
Comment (czue) [inline custom/icds_reports/static/icds_reports/js/spec/download.directive.spec.js]: one little thought is that it'd be nice if changing these three values could be done by only changing one value. so something like:

`var numberOfReports = 12`

as a variable scoped to the whole test and then `assert.equal(numberOfReports - 1, length);` etc. (not sure if that's the right variable name as I don't have the full context on these tests)

**Review by czue — APPROVED** [review]

File: custom/icds_reports/templates/icds_reports/icds_app/download.directive.html (hunk 2 above)
Comment (Rohit25negi) [reply custom/icds_reports/templates/icds_reports/icds_app/download.directive.html]: Yeah right

File: custom/icds_reports/static/icds_reports/js/spec/download.directive.spec.js (hunk 3 above)
Comment (Rohit25negi) [reply custom/icds_reports/static/icds_reports/js/spec/download.directive.spec.js]: that makes sense to me.

**Pull request merged**